example/rollup/app.output.js

```diff
@@ -0,0 +1,19 @@
+// https://github.com/maxogden/websocket-stream/blob/48dc3ddf943e5ada668c31ccd94e9186f02fafbd/ws-fallback.js
+
+var ws = null;
+
+if (typeof WebSocket !== 'undefined') {
+  ws = WebSocket;
+} else if (typeof MozWebSocket !== 'undefined') {
+  ws = MozWebSocket;
+} else if (typeof global !== 'undefined') {
+  ws = global.WebSocket || global.MozWebSocket;
+} else if (typeof window !== 'undefined') {
+  ws = window.WebSocket || window.MozWebSocket;
+} else if (typeof self !== 'undefined') {
+  ws = self.WebSocket || self.MozWebSocket;
+}
+
```

Comment:
should we throw an error when no WebSocket detected, like `ws` did: https://github.com/websockets/ws/blob/master/browser.js

Reply:
can better use "globalThis"?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis
especially since most modern browsers work from a regular WebSocket
https://caniuse.com/mdn-api_websocket